### ISSUE TITLE
M - ENHANCEMENT - Provide a way to resend notifications 

### ISSUE THREAD

**Issue by atl2017** [smap-consulting/smapserver#150]
### Expected Behaviour: Provide a way to resend notification when it has been an error:

![image](https://user-images.githubusercontent.com/29519901/33628369-80edb824-d9ce-11e7-872d-120837d67baf.png)

**Comment by atl2017** [comment]
Neil,

It is really frustrating for my customers to see that Emails was not sent but it is not possible to do anything to fix it and repeat the process. Maybe at least to have chance to bulk downloading of pdf: #99

![image](https://user-images.githubusercontent.com/29519901/35179939-e6fb6978-fd72-11e7-9270-bb6fe0dbf38a.png)

**Comment by nap2000** [comment]
The solution to this will not be backwardly compatible.  Ie you will only be able to resend failed notifications created after the update.

**Comment by atl2017** [comment]
Understood

**Comment by atl2017** [comment]
Thanks.

I supossed "undefined" is due a transition in history, ..

![image](https://user-images.githubusercontent.com/29519901/35383776-38d90e4a-0191-11e8-9a31-12a9684197a9.png)

**Comment by nap2000** [comment]
Is that not what you are expecting?

**Comment by atl2017** [comment]
I am refering to status column.  I guessed that when a notification had a problem a link would be displayed in Retry to resend. Yes it will be OK.

Is it possible to include Form Name (and Instance Name) to see what record is about it?

**Comment by atl2017** [comment]
Tested Ok:

I saw how is working:

![image](https://user-images.githubusercontent.com/29519901/35412891-bf95836e-01eb-11e8-86f5-d20c75c5953a.png)

But.. Status column does not have the right name. Instead of undefined in green color should be "Sucess" and instead of undefined in red should be "Error".

Also, I introduced an artificial error by misspelling email, but it was marked as success. Why?

![image](https://user-images.githubusercontent.com/29519901/35412829-837c4ec6-01eb-11e8-8ade-25b59b70a76b.png)
